pkg/sentry/fs/proc/task.go

```diff
@@ -355,6 +356,67 @@ func newNamespaceDir(t *kernel.Task, msrc *fs.MountSource) *fs.Inode {
 	return newProcInode(t, d, msrc, fs.SpecialDirectory, t)
 }
 
+// memData implements /proc/[pid]/mem.
+type memData struct {
```

Comment:
If we have an implementation here, we'll want an analogous implementation for VFSv2. See https://github.com/google/gvisor/tree/master/pkg/sentry/fsimpl/proc

Reply:
This is implemented now.

Reply:
@ianlewis Do you know someone who could take a look/review the VFSv2 implementation? This would be great so I can get this approved.